addons/project_issue/models/project_issue.py

```diff
@@ -0,0 +1,324 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from openerp import _, api, fields, models, tools
```

Comment:
Import should be alphabetical order like : 
from openerp import api, fields, models, tools, _